packages/documentation/docs/installation/installation.md

```diff
@@ -23,4 +24,11 @@ Follow the links below to the respective installation guides:
   <Card link="installation/javascript"><WebComponentsIcon class="Card_Icon" />Web Components</Card>
   <Card link="installation/vue"><VueIcon class="Card_Icon" />Vue&nbsp;<span style={{fontSize: '0.8rem'}}>(experimental)</span></Card>
   <Card link="installation/blazor"><BlazorIcon class="Card_Icon" />Blazor&nbsp;<span style={{fontSize: '0.8rem'}}>(experimental)</span></Card>
-</CardList>
+</CardList>  
+
+   
+**Experience the Industrial Experience style while coding**  
```

Comment:
after talking to @tokyojen plz add "Siemens" (Experience the Siemens Industrial ...)